sympy/plotting/plot.py

```diff
@@ -1730,6 +1730,26 @@ def plot3d_parametric_surface(*args, **kwargs):
     return plots
 
 
+def conplot(*args, **kwargs):
+    """
+    Draws contour plot of a given function.
+    """
+    args = list(map(sympify, args))
+    symbols = set()
+    show = kwargs.pop('show', True)
+    plot_expr = check_arguments(args, 1, 2)
+    symbols |= plot_expr[0].free_symbols
```

Comment:
What is the point of this. You set `symbols` to an empty set and then you set it to `free_symbols`. Why not just check `free_symbols`? Did you intend to write `symbols = kwargs.pop('symbols', set())`?